libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/columnChart/PluggableColumnChart.tsx

```diff
@@ -19,4 +24,129 @@ export class PluggableColumnChart extends PluggableColumnBarCharts {
     public getSupportedPropertiesList(): string[] {
         return COLUMN_CHART_SUPPORTED_PROPERTIES[this.axis || AXIS.DUAL] || [];
     }
+
+    protected getDefaultAndAvailableSort(
+        measures: IBucketItem[],
+        viewBy: IBucketItem[],
+        stackBy: IBucketItem[],
+        canSortStackTotalValue: boolean,
+    ): {
+        currentSort: ISortConfig["currentSort"];
+        availableSorts: ISortConfig["availableSorts"];
+    } {
+        if (viewBy.length === 2) {
+            if (measures.length >= 2 && !canSortStackTotalValue) {
+                return {
+                    currentSort: [
+                        newAttributeSort(viewBy[0].localIdentifier, "asc"),
+                        newAttributeSort(measures[0].localIdentifier, "asc"),
```

Comment:
measure localIdentifier used for creation of attributeSort?
should not it be viewBy[1].localIdentifier?